packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -1531,6 +1547,24 @@ export class Table extends TableBase {
     }));
   }
 
+  private setTimeToLive(timeToLiveAttribute?: string) {
```

Comment:
`setTimeToLive()` is not a great name for this function.

How about something like `createTimeToLiveCustomResource()`?

Reply:
Sounds good to me. I have changed it to `createTimeToLiveCustomResource(...)`.